src/Apps/Order2/Routes/Checkout/Mutations/useOrder2AddInitialOfferMutation.ts

```diff
@@ -6,30 +6,28 @@ export const useOrder2AddInitialOfferMutation = () => {
   return useMutation<useOrder2AddInitialOfferMutationType>({
     mutation: graphql`
       mutation useOrder2AddInitialOfferMutation(
-        $input: CommerceAddInitialOfferToOrderInput!
+        $input: createBuyerOfferInput!
       ) {
-        commerceAddInitialOfferToOrder(input: $input) {
-          orderOrError {
-            ... on CommerceOrderWithMutationSuccess {
+        createBuyerOffer(input: $input) {
+          offerOrError {
+            ... on OfferMutationSuccess {
               __typename
-              order {
+              offer {
                 internalID
-                mode
-                totalListPriceCents
-                ... on CommerceOfferOrder {
-                  myLastOffer {
-                    internalID
-                    amountCents
-                    note
-                  }
+                amount {
+                  display
+                }
+                note
```

Comment:
Since the step should close success, I think these lines could be removed and we could instead catch the change in the component props to update [these form state values](https://github.com/artsy/force/pull/16310/files). Quite possible we don't need to update them at this point at all - the `submitted` value is just what is coming back as saved to the order.